docs/ide-setup.md

```diff
@@ -17,94 +17,111 @@ Below you find some instructions on how to setup your preferred IDE for CosmoSco
-  - _Working directory_: `<path to cosmoscout>/install/linux-Debug/bin`
-  - _Before launch_ -> __+__ -> _Install_
+  - **Target**: `cosmoscout`
+  - **Executable** -> _Select other_ -> `<path to cosmoscout>/install/linux-Debug/bin/cosmoscout`
+  - **Program arguments**: `--settings=../share/config/simple_desktop.json -vista vista.ini`
+  - **Environment variables**: `LD_LIBRARY_PATH=../lib:../lib/DriverPlugins:$LD_LIBRARY_PATH;VISTACORELIBS_DRIVER_PLUGIN_DIRS=../lib/DriverPlugins`
+  - **Before launch** -> __--__ -> _Build_
+  - **Before launch** -> __+__ -> _Install_
 
 ### Windows
 
-Currently CLion only supports debugging with MinGW and Cygwin. Sadly CosmoScout only runs on MSVC so only a release 
-build will be possible for now.
-Following this restriction you need to install Visual Studio beforehand. If you are not bound to a specific version use
-the latest Visual Studio.
+You need to install Visual Studio beforehand. If you are not bound to a specific version use the latest Visual Studio.
 
 The following will use `-G "Visual Studio 16 2019" -A x64` as the compilation target, you can replace it with your 
 highest available (and supported) version.
 
+- (_optional_) If you want a debug build run `set COSMOSCOUT_DEBUG_BUILD=true`
 - Run: `.\make_externals.bat -G "Visual Studio 16 2019" -A x64`
-- Run: `.\make.bat -G "Visual Studio 16 2019" -A x64`
 - Go to: _Settings_ -> _Build, Execution, Deployment_ -> _Toolchain_
   - Add Visual Studio if it is not in the list yet. And make sure it is at the top.
   - If Visual Studio is installed correctly everything else here should be set automagically.
 - Go to: _Settings_ -> _Build, Execution, Deployment_ -> _CMake_
 - Release Profile
-  - _Generation path_: `build\windows-Release`
-  - _Build options_: `--parallel 8`
-  - _CMAKE options_:
+  - **Build Type**: `Release`
+  - **Generation path**: `build\windows-Release`
+  - **Build options**: `--parallel <your core count>`
+  - **Environment**: If you haven't set `BOOST_ROOT` as a system wide environment variable you need to set it here!
+  - **CMake options**:
     ```
-    -DCMAKE_BUILD_TYPE=Release
     -G "Visual Studio 16 2019" -A x64
     -DCMAKE_INSTALL_PREFIX="<path to cosmoscout>\install\windows-Release"
     -DCOSMOSCOUT_EXTERNALS_DIR="<path to cosmoscout>\install\windows-externals-Release"
     ```
 
+- Debug Profile
+  - **Build Type**: `Debug`
+  - **Generation path**: `build\windows-Debug`
+  - **Build options**: `--parallel <your core count>`
+  - **Environment**: If you haven't set `BOOST_ROOT` as a system wide environment variable you need to set it here!
+  - **CMake options**:
+    ```
+    -G "Visual Studio 16 2019" -A x64
+    -DCMAKE_INSTALL_PREFIX="<path to cosmoscout>\install\windows-Debug"
+    -DCOSMOSCOUT_EXTERNALS_DIR="<path to cosmoscout>\install\windows-externals-Debug"
+    ```
+
 - Go to: _Run/Debug Configuration_ -> _CMake Application_ -> _cosmoscout_
```

Comment:
Should here be an info that the exe is only there if the app has been build? Or is this implied? 😄

Reply:
I added an instruction to run the install task first.